alt-ergo-lib.opam.template

```diff
@@ -0,0 +1,8 @@
+# This part comes from the template. Please edit alt-ergo.opam.template
+# and not alt-ergo-lib.opam which is generated by dune
+
+build: [
+  ["./configure" name]
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
```

Comment:
You should keep the in this template the part the dune auto-generated in the commits before, that dealt with the `{with-doc}` part at least I think (the test part is much less useful because we do not run the tests through dune and we'd need to discuss whether the non-regressions tests should be run by opam).